AirCasting/SessionViews/MeasurementsView.swift

```diff
@@ -0,0 +1,104 @@
+// Created by Lunar on 07/06/2021.
+//
+
+import SwiftUI
+
+struct ABMeasurementsView: View {
+    
+    @ObservedObject var session: SessionEntity
+    var threshold: SensorThreshold
+    @Binding var selectedStream: MeasurementStreamEntity?
+    
+    private var streamsToShow: [MeasurementStreamEntity] {
+        let allStreams = [session.pm1Stream,
+                          session.pm2Stream,
+                          session.pm10Stream,
+                          session.FStream,
+                          session.HStream]
+        
+        let toShow = allStreams.compactMap { $0 }
+        return toShow
+    }
+    private var hasAnyMeasurements: Bool {
+        streamsToShow.filter { $0.latestValue != nil }.count > 0
+    }
+    
+    var body: some View {
+        if hasAnyMeasurements {
```

Comment:
inverting and using `guard hasAnyMeasurements` would make this code a bit more friendly

Reply:
Guard cannot be used inside SwiftUI view builder.